kong/plugins/acme/schema.lua

```diff
@@ -1,11 +1,24 @@
 local typedefs = require "kong.db.schema.typedefs"
+local reserved_words = require "kong.plugins.acme.reserved_words"
 
 local CERT_TYPES = { "rsa", "ecc" }
 
 local RSA_KEY_SIZES = { 2048, 3072, 4096 }
 
 local STORAGE_TYPES = { "kong", "shm", "redis", "consul", "vault" }
 
+local function validate_namespace(namespace)
+  if namespace ~= "" then
+    for _, v in pairs(reserved_words) do
+      if namespace:sub(1, #v) == v then
```

Comment:
should we use `pl.startswith(s, prefix)`?

Reply:
Or `namespace:find(v, 1, true) == 1`?

Reply:
Not sure which one is more efficient. `string.sub` generates a new string while `string.find` may need to traverse the whole string.

Reply:
OK, let's keep this line.